### ISSUE TITLE
mass update bug

### ISSUE THREAD

**Issue by Wjxfi** [Droid-ify/client#286]
If you close the application during a mass update in the legacy mode, the android dialog will no longer be shown (for downloading apps) although the notification says that the application is downloading. Even if you go back to the app, the "install?" dialog will no longer appear. For all other apps since closing droid-ify. corrected by pressing the "download all updates" button again.

miui 13.0.5, android 12, 0.5.6 izzyondroid

**Comment by Iamlooker** [comment]
Yes, I saw this. But your last issue has been resolved in  new release so you can use Session Installer with it in meanwhile.

**Comment by Elbullazul** [comment]
Had this happen to me too on v5.0.7. Video of the issue: https://streamable.com/cpabpg

Clicking on the "download is complete, click to install" notification also soft-locks the install button

Closing and reopening Droid-ify allows the "update all" button to work normally

**Comment by Iamlooker** [comment]
I will look into this issue after new format is ready
